feature/system/system_base_test/tests/system_g_protocol_test/system_g_protocol_test.go

```diff
@@ -40,29 +40,38 @@ func TestMain(m *testing.M) {
 	fptest.RunTests(m)
 }
 
-func dialConn(t *testing.T, dut *ondatra.DUTDevice, svc introspect.Service, wantPort uint32) *grpc.ClientConn {
+func dialConn(t *testing.T, dut *ondatra.DUTDevice, svc introspect.Service, wantPort uint32, nonStandardPort bool) *grpc.ClientConn {
```

Comment:
The parameter `nonStandardPort` is not used?
```suggestion
func dialConn(t *testing.T, dut *ondatra.DUTDevice, svc introspect.Service, wantPort uint32) *grpc.ClientConn {
```

Reply:
Updated if `!deviations.NonStandardGRPCPort(dut) {`
to `if !nonStandardPort {` in dialConn function.